microcosm_caching/tests/test_decorators.py

```diff
@@ -64,12 +64,9 @@ def setup(self):
         self.cache_prefix = "test"
         controller = self.graph.controller
 
-        self.cached_retrieve = cached(controller, TestSchema, self.cache_prefix)(controller.retrieve)
-        self.cached_extended_retrieve = cached(
-            controller,
-            TestExtendedSchema,
-            self.cache_prefix,
-        )(controller.extended_retrieve)
+        self.cached_retrieve = cached(controller, TestSchema)(controller.retrieve)
+        self.cached_extended_retrieve = cached(controller, TestExtendedSchema)(controller.extended_retrieve)
+        self.cached_retrieve_for = cached(controller, TestForSchema)(controller.retrieve_for)
```

Comment:
The remaining tests shouldn't need to be changed, the cache_prefix above mirrors the value used when initializing the graph, so any mismatches in key calculation would be caught here.